config/optional/user.role.authenticated.yml

```diff
@@ -0,0 +1,26 @@
+langcode: en
```

Comment:
You should not define this role here since it is already defined in the Drupal core. Instead, you can modify it.